app/controllers/games_controller.rb

```diff
@@ -0,0 +1,20 @@
+class GamesController < ApplicationController
```

Comment:
So if I add an Update method right now, all that can be updated is the game :name. I would assert there isn't a reason to allow the name to be updated, and in that case, I can't think what we would need an Update method for...?